backend/src/routes/account.js

```diff
@@ -16,6 +14,20 @@ router.get('/',
     res.send(req.user)
   })
 
+router.patch('/', async (req, res, next) => {
+  if (!req.user) return
+
+  const { director } = req.body
+
+  if (!director) res.sendStatus(400)
+
+  const user = await User.findByIdAndUpdate(req.session.userId, { director })
+
+  req.session.user = user
+
+  res.status(204).send(req.user)
```

Comment:
we can send `user` instead of `req.user`

Reply:
done